e2e/support/test_staging_hooks.go

```diff
@@ -30,16 +30,14 @@ import (
 )
 
 func init() {
```

Comment:
Probably the entire init() func can be removed to avoid the build error.

Reply:
I have removed init function, Is it correct now?

Reply:
Thanks. However I noticed I was wrong. In the diff code snippet I did not see that the func is also having a second part with more logic. We need to restore it and just get rid of the build error. In order to verify that, you should try to build locally before submitting the change. Just do `make build` and you should compile successfully before submitting the PR.